content/_data/courses.json

```diff
@@ -1,5 +1,14 @@
 {
   "courses": [
+    {
+      "title": "Boas práticas em automação de testes com Cypress",
+      "url": "https://www.udemy.com/course/boas-praticas-em-automacao-de-testes-com-cypress",
+      "author": "Walmyr Lima e Silva Filho",
+      "authorTwitter": "walmyrlimaesilv",
+      "sourceName": "Udemy",
+      "sourceUrl": "https://udemy.com"
```

Comment:
```suggestion
      "sourceUrl": "https://udemy.com",
```

Reply:
@wlsf82 it looks like you forgot to add a comma here, and the json in therefore invalid. This may be the cause of the issue. Hopefully with this simple change all the tests will now pass :)

Reply:
🤦🏻 Oh, thanks for fixing it!